src/containers/App.js

```diff
@@ -47,48 +51,29 @@ function App() {
 
   return (
     <div className="main">
-      <div className="main__search-panel">
-        <SearchForm onFinish={onFinish}/>
-        <div className="main__selector-menu">
-          <p>Категории</p>
-          <CategorySelector 
-            onFinish={onFinish}
-            bookTitle={bookTitle} 
-            sort={sort} 
-          />
-          <p>Сортировать по</p>
-          <SortSelector 
-            onFinish={onFinish}
-            bookTitle={bookTitle} 
-            category={category} 
-          />
-        </div>
-        <p>Всего найдено {resultsValue} книг</p> 
-      </div>
+      <SearchPanel
+        onFinish={onFinish}
+        bookTitle={bookTitle}
+        sort={sort}
+        category={category}
+        resultsValue={resultsValue}
+      />
       <div className="main__content">
       {
-        data.map(book => {
-          if (!book.volumeInfo.imageLinks) {
-            book.volumeInfo.imageLinks = {};
-            book.volumeInfo.imageLinks.thumbnail = 'https://riossport.ru/local/templates/riossport/assets/images/no-image.png';
-          }
-          if (!book.volumeInfo.categories) {
-            book.volumeInfo.categories = 'No categories';
-          }
-          if (!book.volumeInfo.authors) {
-            book.volumeInfo.authors = 'No authors';
-          }
-          return book;
-        }).map(book => (
-        <BookItem
-          key={book.id + book.etag}
-          imageLink={book.volumeInfo.imageLinks.thumbnail}
-          category={book.volumeInfo.categories}
-          bookTitle={book.volumeInfo.title}
-          author={book.volumeInfo.authors}
-        />
-      ))
+        data.map(book => (
+          <BookItem
+            key={book.id + book.etag}
+            thumbnail={book.volumeInfo?.imageLinks?.thumbnail}
+            category={book.volumeInfo?.categories}
+            bookTitle={book.volumeInfo.title}
+            author={book.volumeInfo?.authors}
```

Comment:
why do you use 4 props with 1 object book? just set one prop with link to this object